### PR TITLE
Intsight indicators fix.

### PR DESCRIPTION
**Pull request by orlichter1**
## Status
Ready

## Related Issues
Redoing of https://github.com/demisto/content/pull/4219 due to changes while waiting for pr.
related to: https://github.com/demisto/etc/issues/17949

## Description
Fixing indicator outputs in accordance with standards.

## Required version of Demisto
x.x.x

## Does it break backward compatibility?
   - No

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [ ] Code Review

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](https://github.com/demisto/content/edit/Intsight-indicators2/Integrations/IntSight/IntSight.yml?pr=%2Fdemisto%2Fcontent%2Fpull%2F4483)
- [ ] [CHANGELOG](https://github.com/demisto/content/edit/Intsight-indicators2/Integrations/IntSight/CHANGELOG.md?pr=%2Fdemisto%2Fcontent%2Fpull%2F4483)

### CODE REVIEW

**Comment by orlichter1** [comment]
@kirbles19 All we changed is the indicators outputs to be according to standards. Do we need to regenerate documentation?

File: Integrations/IntSight/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ## [Unreleased]
-
+- Indicators issue fixed - were not extracted correctly in `!intsights-get-iocs`
```
Comment (yaakovi) [inline Integrations/IntSight/CHANGELOG.md]: maybe:
```suggestion
- Fixed an issue where indicators were not extracted correctly in ***intsight-get-iocs*** command
```
@kirbles19 WDYT?

Comment (kirbles19) [reply Integrations/IntSight/CHANGELOG.md]: Add a period to the end, and it's perfect.

Comment (yaakovi) [reply Integrations/IntSight/CHANGELOG.md]: so close...

**Comment by richardbluestone** [comment]
The yml file has been reviewed and updated.

**Pull request merged**